src/components/sbb-journey-summary/sbb-journey-summary.scss

```diff
@@ -21,15 +21,14 @@
 
   margin-block-start: 0;
   margin-block-end: 0;
-  padding-inline-start: 0;
```

Comment:
I think those changes should not appear hear. Did you maybe forgot to rebase on to lit-migration branch? (there was an update to journey summary last week)

Reply:
Not a mistake, I had to fix a spacing issue between <span> due to a difference between JSX and lit rendering